examples/large_models/Huggingface_pippy/Readme.md

```diff
@@ -46,37 +39,42 @@ pippy:
     input_names: ['input_ids'] # input arg names to the model, this is required for FX tracing
     model_type: "HF" # set the model type to HF if you are using Huggingface model other wise leave it blank or any other model you use.
     rpc_timeout: 1800
+    num_worker_threads: 512 #number of threads for rpc worker usually 512 is a good number
 
 handler:
     max_length: 80 # max length of tokens for tokenizer in the handler
+    model_name: "/home/ubuntu/serve/examples/large_models/Huggingface_pippy/model/models--facebook--opt-30b/snapshots/ceea0a90ac0f6fae7c2c34bcb40477438c152546" #the path to the checkpoints, in this example downloaded file
+    index_file_name: 'pytorch_model.bin.index.json' # index json file name in the model checkpoint folder, that keeps information of distributed checkpoints
+    manual_seed: 40
+    dtype: fp16 # data type to load your model checkpoint, supported fp32, fp16, bf16
 ```
 
 ### Step 3: Generate Tar/ MAR file
 
 Navigate up to `Huggingface_Largemodels` directory.
 
 ```bash
-torch-model-archiver --model-name bloom --version 1.0 --handler pippy_handler.py --extra-files model/models--facebook--opt-iml-max-1.3b/snapshots/d60fa58f50def19751da2075791da359ca19d273  -r requirements.txt --config-file model-config.yaml --archive-format tgz
+torch-model-archiver --model-name opt --version 1.0 --handler pippy_handler.py  -r requirements.txt --config-file model-config.yaml --archive-format tgz
```

Comment:
Will be good to clarify that in this case since the model file is large we are using the model-config file to point to location of the file so that .gz does not copy over entire model again and simplifies the developer testing. For production settings, the large models can be put in some shared location and used from there in the model-config. We can create a separate doc page describing the details for "large models" and point to that.